packages/design-system/src/components/TextField/style.scss

```diff
@@ -38,4 +45,10 @@ $base-class: 'text-field';
     align-items: center;
     justify-content: flex-start;
   }
+  &__label-charsCounter {
```

Comment:
<img width="343" alt="Design System - LiveChat API Developers Docs and Guides 2021-05-04 12-05-17" src="https://user-images.githubusercontent.com/24607370/116988999-7706aa80-acd1-11eb-8cd7-8570de019e22.png">
I aksed Jo about this - both labelAdornment and new element should be positioned with `flex-end`